litmus-portal/frontend/src/pages/MyHub/index.tsx

```diff
@@ -70,47 +85,86 @@ const MyHub = () => {
-                            }}
-                          >
-                            <CardContent className={classes.cardContent}>
-                              <img
-                                src="/icons/my-hub-charts.svg"
-                                alt="add-hub"
-                              />
-                              <Typography
-                                variant="h6"
-                                align="center"
-                                className={classes.hubName}
-                              >
-                                {hub.HubName}
-                              </Typography>
-                              <Typography
-                                variant="h6"
-                                align="center"
-                                className={classes.hubBranch}
-                              >
-                                {hub.GitURL.split('/')[4]}/{hub.GitBranch}
-                              </Typography>
-                            </CardContent>
-                          </Card>
-                        );
-                      })}
+                      {totalHubs &&
+                        totalHubs.map((hub: HubDetails) => {
+                          return (
+                            <Paper
+                              key={hub.id}
+                              elevation={3}
+                              className={classes.cardDivChart}
+                            >
+                              <CardContent className={classes.cardContent}>
+                                <Typography
+                                  className={
+                                    hub.IsAvailable
+                                      ? classes.connected
+                                      : classes.error
+                                  }
+                                >
+                                  {hub.IsAvailable ? 'Connected' : 'Error'}
+                                </Typography>
+                                <img
+                                  src="/icons/my-hub-charts.svg"
+                                  alt="add-hub"
+                                />
+                                <Typography
+                                  variant="h6"
+                                  align="center"
+                                  className={classes.hubName}
+                                >
+                                  {hub.HubName}
+                                </Typography>
+                                <Typography
+                                  variant="h6"
+                                  align="center"
+                                  className={classes.hubBranch}
+                                >
+                                  {hub.RepoURL.split('/')[4]}/{hub.RepoBranch}
+                                </Typography>
+                                <Typography style={{ fontSize: '12px' }}>
```

Comment:
Please consider including this in the styles.ts file. Also please try using 'rem' units.